.gitignore

```diff
@@ -34,3 +34,5 @@ debug
 resource.syso
 
 *.resource.go
+
+vendor
```

Comment:
If the building is happening in a temporary directory, do we need to ignore vendor dir?

Reply:
yeah, maybe its a good idea to do everything in tmp dir.

Done.